1_Installation_Files (v1.5.5)/YOUR_ADMIN/image_handler.php

```diff
@@ -484,7 +484,7 @@ function popupWindow(url) {
                     <td class="dataTableContent ih-center"><?php echo $text_base_size; ?></td>
 <?php
             $preview_image = $tmp_image_preview->get_resized_image(IMAGE_SHOPPING_CART_WIDTH, IMAGE_SHOPPING_CART_HEIGHT, 'generic');
-            list($width, $height) = @getimagesize(DIR_FS_CATALOG . $preview_image);
+            [$width, $height] = getimagesize(DIR_FS_CATALOG . $preview_image);
```

Comment:
This array-destructuring syntax only works since PHP 7.1
https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring

Reply:
Did this get changed back?  I don't mind having the minimum PHP version at 5.6, but 7.1 is pushing it.